tests/settings_tests/tests.py

```diff
@@ -445,3 +447,25 @@ def test_tuple_settings(self):
             finally:
                 del sys.modules['fake_settings_module']
                 delattr(settings_module, setting)
+
+
+class TestDatabaseMirror(unittest.TestCase):
```

Comment:
You're writing to the database. You need those changes to be reset. You need to be using Django's `TestCase`, rather than `unittest.TestCase`.